Editor/ProjectAuditorSettingsProvider.cs

```diff
@@ -106,9 +106,10 @@ public void SelectCurrentSettings(ProjectAuditorSettings settings)
             m_CurrentSettings = settings;
 
             var path = AssetDatabase.GetAssetPath(settings);
-            var guid = AssetDatabase.GUIDFromAssetPath(path);
+            var guidAsString = AssetDatabase.AssetPathToGUID(path);
 
-            var guidAsString = path == "" || guid.Empty() ? "" : guid.ToString();
+            if (!string.IsNullOrEmpty(guidAsString))
+                UserPreferences.settingsAsset = guidAsString;
             UserPreferences.settingsAsset = guidAsString;
```

Comment:
I see that this line matches the conditional line just above.

So the intention here was:  If the GUID is not valid (the else-case basically) set the preferences to empty string "" to signal the fallback to default (in-memory) settings.

Reply:
You ended up making the same change I had locally :-)
Feel free to approve and merge.